pkg/static/login.js

```diff
@@ -338,6 +338,27 @@ import "./login.scss";
         event.stopPropagation();
     }
 
+    function deal_with_multihost() {
+        // If we are currently logged in to some machine, but still
+        // end up on the login page, we are about to load resources
+        // from two machines into the same browser origin. This needs
+        // to be allowed explicitly via a configuration setting.
+
+        const logged_into = environment.logged_into || [];
+        const cur_machine = logged_into.length > 0 ? logged_into[0] : null;
```

Comment:
`const cur_machine = environment.logged_into?.[0]`

(but if you prefer this, it's fine)

Reply:
Looks too much like Perl for my taste... :-) but let's do it.